charts/pega/templates/pega-srs-mtls-secret.yaml

```diff
@@ -0,0 +1,20 @@
+{{ if ((eq (include "deployNonExtSRSSecret" .) "true")) }}
+kind: Secret
+apiVersion: v1
+metadata:
+  name: {{ template "pega-srs-mtls-secret-name" $ }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-weight": "0"
+    "helm.sh/hook-delete-policy": before-hook-creation
+data:
+  {{ if .Values.pegasearch.srsMTLS.keyStorePassword -}}
+  # Base64 encoded password for the SRS trust store
+  SRS_KEYSTORE_PASSWORD: {{ .Values.pegasearch.srsMTLS.keyStorePassword | b64enc }}
```

Comment:
In general, secret content is encoded by k8s. Why is it required to encode again?